util/debug.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import "os"
+
+func IsDebugEnabled() bool {
+	val, exists := os.LookupEnv("debug")
+	return exists && (val == "1" || val == "true")
+}
+
+func UseDockerCLI() bool {
```

Comment:
Beta testing: Default to Docker CLI to keep current behaviour, faas_docker_cli=0 to use Docker API